dask/dataframe/tests/test_multi.py

```diff
@@ -1406,6 +1406,10 @@ def test_concat_unknown_divisions():
     with pytest.raises(ValueError):
         dd.concat([aa, cc], axis=1)
 
+    with warnings.catch_warnings(record=True) as w:
+        dd.concat([aa, bb], axis=1, ignore_unknown_divisions=True)
+        assert len(w) == 0
```

Comment:
For uniformity, this should use [`pytest.warns`](https://docs.pytest.org/en/latest/warnings.html#recwarn) instead:

```suggestion
    with pytest.warns(None) as record:
        dd.concat([aa, bb], axis=1, ignore_unknown_divisions=True)
    assert len(record) == 0
```